x-pack/plugins/observability_solution/asset_manager/server/routes/enablement/install_builtin_definitions.ts

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
nit: i wonder if these files should live outside 'enablement' directory, since they are not in that url path?